packages/cli/src/constructs/heartbeat-check.ts

```diff
@@ -0,0 +1,68 @@
+import { Check, CheckProps } from './check'
+import { Session } from './project'
+import { DateTime } from 'luxon'
+
+type unitType = 'seconds' | 'minutes' | 'hours' | 'days'
```

Comment:
Maybe `type TimeUnits` for the name is more specific

Reply:
Good call!